cluster/gce/config-default.sh

```diff
@@ -140,6 +140,10 @@ export AGGREGATOR_MASTER_NAME="${INSTANCE_PREFIX}-aggregator"
 export INITIAL_ETCD_CLUSTER="${MASTER_NAME}"
 export MASTER_TAG="${INSTANCE_PREFIX}-master"
 export NODE_TAG="${INSTANCE_PREFIX}-minion"
+# ETCD_LISTEN_CLIENT_IP decides whether etcd servers should also listen on host IP, 
```

Comment:
You may want to have the same thing in config-test.sh ?

Reply:
Added the variable into config-test.sh